x/bridge/README.md

```diff
@@ -144,6 +155,27 @@ _Detailed description is to be done._
 
 ![observer](images/observer.png)
 
+### Quarantine
+
+The **quarantining** might be initiated right after an inbound transfer reaches the required number of votes for processing. When a transfer receives the final necessary vote, the module evaluates it against the asset's rate limits. If the transfer exceeds these limits, it is added to the end of the quarantine queue. This queue can be activated for processing through a governance proposal. Notably, **quarantining** allows for partial execution: if a part of the transfer falls within the rate limits, that portion will be minted, while the remainder goes into the queue.
+
+There are two methods for managing quarantined assets:
+
+* Simply storing the queue in the state. Here, if rate limits are exceeded, the coins are added to the state queue without minting. Upon release, the tokenfactory's Mint function is called for the relevant addresses.
+* Minting all coins to a special **quarantine** address (not the destination address) immediately after finalization. The coins are then transferred to their intended addresses upon release following a governance proposal. This method still requires the state queue, but operations are based solely on straightforward transfers. It also offers a transparent view of the **quarantine**, represented by a dedicated **quarantine** vault on Osmosis.
+
+![quarantine](images/quarantine.png)
+
+For **quarantining**, a queue is utilized instead of a map to allow sequential execution of transfers and to potentially filter out malicious transactions. While this queue cannot eliminate all malicious transfers, it can help segregate them and make them less harmful. As a security measure, periodically matching the total supply of minted assets with the balances in external vaults can help identify fraudulent activity.
+
```

Comment:
What do you think about this scenario

BTC Height 10 comes in, gets added to quarantine
BTC Height 20 comes in, gets added to quarantine after the entry from height 10
BTC Height 15 comes in, in your description, this would get added after 20. Is this desired?

Reply:
It won't be a problem, the order of transfers doesn't play any role. I added it to the doc along with the example